doc/code/qml_fourier.rst

```diff
@@ -87,10 +87,71 @@ available to a quantum model, the larger the class of functions that model can
 represent, potentially leading to greater utility for quantum machine learning
 applications.
 
+Calculating the frequencies supported by an encoding
+----------------------------------------------------
+
+For certain circuits, information on the frequency spectra :math:`\Omega_i`
+can solely be derived from the structure of the gates
+that encode the corresponding inputs :math:`x_i` (see for example `Schuld,
+Sweke and Meyer (2020) <https://arxiv.org/abs/2008.08605>`__). More precisely, if
+all input-encoding gates are of the form :math:`e^{-ix_i G}`, where :math:`G` is
```

Comment:
Recommend to change notation to use x_j and Omega_j (or some other subscript) in this section so as not to clash with the i in e^{-i x_j G}.